browser/src/map/Map.js

```diff
@@ -347,6 +351,31 @@ L.Map = L.Evented.extend({
 
 	// end of A11y
 
+	// can be '', 'SAVING', 'MODIFIED' or 'SAVED'
+	setDocumentStatus: function(status) {
```

Comment:
this could be moved to StatusBar.js as a handler for a `fire` signal on last mod status update